src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.commands;
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import seedu.address.model.Model;
+
+/**
+ * Sorts all persons in the address book by the number of tasks completed.
```

Comment:
```suggestion
 * Sorts all students in the address book by the number of tasks completed.
```
In addition, might want to change the description since we are sorting by incomplete tasks I think.